content/en/user-guide/aws/sns/index.md

```diff
@@ -445,6 +446,15 @@ awslocal sns get-subscription-attributes --subscription-arn "arn:aws:sns:us-east
 
 SNS will now publish messages to your HTTP endpoint, even if it did not confirm itself the subscription. 
 
+## Filter operations
+
+LocalStack supports the following SNS filter operators:
+
+- [Equals-ignore-case matching](https://docs.aws.amazon.com/sns/latest/dg/string-value-matching.html#string-equals-ignore)
+- [Suffix matching](https://docs.aws.amazon.com/sns/latest/dg/string-value-matching.html#string-suffix-matching)
+- [Anything-but matching](https://docs.aws.amazon.com/sns/latest/dg/string-value-matching.html#string-anything-but-matching).
+
+LocalStack also supports the [$or operator](https://docs.aws.amazon.com/sns/latest/dg/and-or-logic.html#or-operator), allowing more complex filter policies.
```

Comment:
I'm not too sure about this: we actually supported a lot of filters already before, and these are just the latest ones we've added to be in parity (AWS must have added them around re-invent 2023). We actually only lack support for one type of operator, which is `cidr`, but we should support all the other ones: https://docs.aws.amazon.com/sns/latest/dg/sns-subscription-filter-policies.html

Do we still want to list all of them and put a limitation for `cidr`, or as we're in parity with AWS here, just let the AWS documentation do its job?

Reply:
I would suggest yes. Lets put a note for the limitation as well, and once we're in parity we can remove this part.

Reply:
I have added this note on current limitation section, and removed this particular section 👍